src/compiler/scanner.ts

```diff
@@ -553,8 +597,13 @@ function isHexDigit(ch: number): boolean {
     return isDigit(ch) || ch >= CharacterCodes.A && ch <= CharacterCodes.F || ch >= CharacterCodes.a && ch <= CharacterCodes.f;
 }
 
-function isCodePoint(code: number): boolean {
-    return code <= 0x10FFFF;
+function isASCIILetter(ch: number): boolean {

```

Comment:
Nit: `isAsciiCharacter` or `isASCIICharacter`

Reply:
But this really just cover letters

Reply:
Ah, OK. This is essentially `/[\p{ASCII}&&\p{Letter}]/v` though

Reply:
P.S. IMO `isDigit` should be renamed to `isASCIIDigit` too